RecoMuon/MuonIdentification/plugins/ME0SegmentMatcher.cc

```diff
@@ -164,6 +164,8 @@ void ME0SegmentMatcher::produce(edm::Event& ev, const edm::EventSetup& setup) {
 
       //Remove later
       if (std::abs(thisTrack->eta()) < 1.8) continue;
+      // low pt tracks cannot be propagated to me0
+      if (thisTrack->pt() < 1.0) continue;
```

Comment:
Are you sure about using pt?
The p at eta~2.8 and pt=1 is about 8 GeV;
for the eta ~4 option, if it resurfaces, there is even larger ratio.